game/state/city/vehiclemission.cpp

```diff
@@ -58,17 +58,11 @@ static const std::map<UString, std::pair<unsigned, unsigned>> selfDestructTimer
 } // namespace
 
 FlyingVehicleTileHelper::FlyingVehicleTileHelper(TileMap &map, Vehicle &v)
```

Comment:
Can all these args be const? And presumably the members variablestoo

Reply:
Can't make `TileMap` reference a const, since it drags too much refactoring in places unrelated to the PR (in `CanEnterTileHelper` and all of its descendants, in `Tile`, etc.).

I guess I will try argument constness refactoring through out the code in another PR.